build-system/tasks/visual-diff/index.js

```diff
@@ -48,9 +48,19 @@ const {waitUntilUsed} = require('tcp-port-used');
 let puppeteer;
 let percySnapshot;
 
+// CSS injected in every page tested.
+// https://docs.percy.io/docs/percy-specific-css
+const percyCss = [
+  // Loader animation may otherwise be captured in slightly different points,
+  // causing the test to flake.
+  '.i-amphtml-new-loader { display: none; }',
```

Comment:
Can we give this a background-color instead of `display: none`? otherwise we won't know the difference between if the loader was excluded versus if it was overridden.

Reply:
How about this?

```suggestion
  '.i-amphtml-new-loader * { animation: none !important; }',
```

Reply:
LGTM

Reply:
+1